contracts/lease/src/api/open/mod.rs

```diff
@@ -196,3 +230,62 @@ mod test_invariant {
         ));
     }
 }
+
+#[cfg(test)]
+mod test_position_spec {
+    use sdk::cosmwasm_std::{from_slice, StdError};
```

Comment:
`sdk` is just another project crate, why it is outside the group of project crates?